src/schema/v2/match/partner.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  GraphQLFieldConfig,
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLString,
+} from "graphql"
+import { ResolverContext } from "types/graphql"
+import Partner from "../partner/partner"
+
+export const PartnerMatch: GraphQLFieldConfig<void, ResolverContext> = {
+  type: new GraphQLList(Partner.type),
+  description: "A Search for Artists",
+  args: {
+    query: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: "Your search term",
+    },
+  },
+  resolve: async (_root, { query }, { matchPartnerLoader }) => {
+    if (!matchPartnerLoader) {
+      return new Error("You need to be signed in to perform this action")
+    }
+
+    const partners = await matchPartnerLoader({ term: query })
+
+    return partners
```

Comment:
Not necessary for this since we're in a searchable endpoint thing! - but heads up, if pagination is ever needed (or more fine-grained control), we use connections for lists, and these docs step through how [to set it up](https://github.com/artsy/metaphysics/blob/main/docs/pagination.md).